build/configs/artik053/artik053_download.sh

```diff
@@ -73,7 +73,7 @@ main()
 				pushd ${OS_DIR_PATH}
 				sh ../tools/fs/mkromfsimg.sh
 				if [ ! -f "../build/output/bin/romfs.img" ]; then
-					echo "ROMFS image is not present"
+					echo "ROMFS image is not present. Install genromfs."
```

Comment:
Why do you add it in here?

Reply:
For notice. If genromfs is not installed executing mkromfsimg.sh could be failed.

Reply:
I think here is not an appropriately location to say this.
How about move it to romfs [[README]](https://github.com/Samsung/TizenRT/blob/master/tools/fs/README_ROMFS.md) as a pre-condition?